contents/tutorials/how-to-run-successful-group-experiments.md

```diff
@@ -0,0 +1,86 @@
+---
+title: How to run successful group-level A/B tests
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: // what to put here?
+featuredImage: //TODO
+tags: ['experimentation']
+---
+
+A/B tests are a powerful tool for measuring how product changes impact user behavior. However, sometimes changing how one person uses your product will affect how others interact with it too. For example, a change in the Uber App for Drivers will likely affect the experience of riders too. In these scenarios, group-level experiments allow us to measure the impact of changes beyond an individual and across a group of users.
+
+// TODO: Add image showing difference between group-level and user-level experiment
+
+## When to run group-level experiments
+
+When the change in how an individual uses your product will greatly affect the behavior of other users, or when you want to measure the impact of a change on an entire group of users. This is usually seen in the following types of products:
```

Comment:
Try to be consistent with what you name one person, so far you use "person", "individual", and "user" when you should probably just use one of these

Reply:
That's a good point. I'll keep it to user as much as possible